react-search-component/my-app/src/SearchableList.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+import './App.css';
+
+export default function SearchableList({ list }) {
+  return (
+    <div>
+      <Input item={list} />
+    </div>
+  );
+}
+
+function Input({ item }) {
+  const [currentLetter, setCurrentLetter] = useState('');
+  const resultItems = item
+    .filter((result) => result.includes(currentLetter) || currentLetter === '')
```

Comment:
You probably want to use `toLowerCase` on both the item and currentLetter so that your search is case insensitive.